source/whats-new.txt

```diff
@@ -39,6 +37,15 @@ Upcoming Breaking Changes
   aggregation framework and removed in a future release.
 - Beginning with v5.0, the {+driver-short+} will require Java 11 or later.
 
+.. _version-4.10:
+
+What's New in 4.10
+------------------
+
+- Interfaces in the ``com.mongodb.client.model.search`` package have the
+  ``@Sealed`` marking instead of ``@Evolving``. Sealed interfaces
```

Comment:
Suggestion for brevity. If you want to stick with the current form, I'd call this an "annotation", not a "marking", to be consistent with the driver.

```suggestion
- Interfaces in the ``com.mongodb.client.model.search`` package are now
  ``@Sealed`` instead of ``@Evolving``. Sealed interfaces
```